src/Symfony/Component/VarDumper/Cloner/VarCloner.php

```diff
@@ -268,7 +273,6 @@ protected function doClone($var)
             }
 
             $queue[$i] = $vals;
-            unset($indexedArrays[$i]);
```

Comment:
another µ-optim: unsetting scalars from arrays does not free memory, but takes CPU, better not doing it